src/IList.ts

```diff
@@ -69,78 +71,78 @@ export interface IList<E> extends DistributedObject {
     get(index: number): Promise<E>;
 
     /**
-     * Returns the index of first occurence of given element in the list.
-     * @param element
-     * @return the index of first occurence of given element or `-1` if the list does not contain given element.
+     * Returns the position of first occurrence of the given element in this list.
+     * @param element element to search for
+     * @return the index of first occurrence of given element or `-1` if the list does not contain given element.
      */
     indexOf(element: E): Promise<number>;
 
     /**
      * Checks if this list is empty.
-     * @return `true` if this list contains no element, `false` otherwise.
+     * @return `true` if this list contains no elements, `false` otherwise.
      */
     isEmpty(): Promise<boolean>;
 
     /**
-     * Returns the index number of last occurence of this element in the list.
+     * Returns position of the last occurrence of the given element in this list.
      * @param element
      */
     lastIndexOf(element: E): Promise<number>;
 
     /**
      * Removes the given element from this list.
-     * @param element
-     * @return `true` if this list changed, `false` otherwise.
+     * @param element element to be removed
+     * @return `true` if this list has changed as a result of this operation, 'false' otherwise.
```

Comment:
code pieces should be surrounded with backtics ( \` ), not single quotes ( \' ).
There are multiple occurences of it in this PR.
It is not a big problem, can be corrected with direct commit later.